scripts/create-aks-acr.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+: "${AKS_NAME:?Environment variable empty or not defined.}"
+: "${ACR_NAME:?Environment variable empty or not defined.}"
+
+register_feature() {
+  # pinning to 0.5.87 because of https://github.com/Azure/azure-cli/issues/23267
+  az extension add --name aks-preview --version 0.5.87
+  # register enable oidc preview feature
+  az feature register --namespace Microsoft.ContainerService --name EnableOIDCIssuerPreview >/dev/null
```

Comment:
This feature is enabled by default in AKS, so this feature registration is no longer required. This also means the `aks-preview` extension is not required.

Reply:
Thanks for calling it out!